### DISCUSSION TITLE
How can I delete invisible release？delete tag but no delete release

### DISCUSSION THREAD

**Discussion by Gabrielxzx** [community/community#106721] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I have delete tag by GitHub CLI , and I did not delete the correspond release.
Now on the website. The number of release is still exist but I cannot view them.
How can I remove these old release?
![image](https://github.com/community/community/assets/27126997/94be80b1-7295-4757-9aab-861f50c34339)

**Comment by Gabrielxzx** [comment]
by the way
I have delete my local repositories，I have try "reflog", But it not work

**Comment by srivarshithdaladuli** [comment, marked as answer]
when a Git tag is deleted outside of GitHub, the associated GitHub releases become invalid but still appear in the interface. To clear things up, you need to manually delete those releases. Go to the Releases page, identify any releases showing "Invalid tag name", and delete them by clicking the Delete button and confirming the deletion.

**Reply by Gabrielxzx** [reply]
Thanks
But Sorry，I cannot understand "identify any releases showing "Invalid tag name"".
The Releases page only show new releases with new tag. 
there is 6 pags， but  I click the "5" or "6", it shows "There aren’t any releases here"
I think those Invalid releases is hide in these page, but i cannot see them

![image](https://github.com/community/community/assets/27126997/a703cea1-49f7-4ea8-b383-66d1f96ae998)

![image](https://github.com/community/community/assets/27126997/bda01cde-d6cc-40cb-874e-a9440b2bd15e)
